core/src/main/java/org/apache/calcite/util/TimeString.java

```diff
@@ -76,35 +76,54 @@ private static String hms(int h, int m, int s) {
     return b.toString();
   }
 
-  /** Sets the fraction field of a {@code TimeString} to a given number
-   * of milliseconds. Nukes the value set via {@link #withNanos}.
+  /**
+   * Sets the fraction field of a {@code TimeString} to a given number of milliseconds.
+   * This will override any fraction value previously set via {@link #withNanos}.
```

Comment:
use present tense, 'This overrides' rather than future tense 'This will override'.